tailwind.config.js

```diff
@@ -53,32 +46,45 @@ module.exports = {
         3: "var(--calcite-ui-border-3)",
         4: "var(--calcite-ui-border-4)",
         5: "var(--calcite-ui-border-5)",
-      }
+      },
+      transparent: "transparent"
     },
     fontFamily: {
-      sans: ["Avenir Next", "Avenir", "Helvetica Neue", "sans-serif"],
-      mono: ["Consolas", "Andale Mono", "Lucida Console", "Monaco", "monospace"],
+      // assets/styles/_type
+      sans: "var(--calcite-sans-family)",
+      mono: "var(--calcite-code-family)",
+      inherit: "inherit"
     },
     fontSize: {
-      "-3": "0.625rem", // 10px
-      "-2": "0.75rem",  // 12px
-      "-1": "0.875rem", // 14px
-      0: "1rem",        // 16px
-      1: "1.125rem",    // 18px
-      2: "1.25rem",     // 20px
-      3: "1.625rem",    // 26px
-      4: "2rem",        // 32px
-      5: "2.5rem",      // 40px
-      6: "3rem",        // 48px
-      7: "3.5rem",      // 56px
-      8: "4rem",        // 64px
+      // assets/styles/_type
+      "-3": "var(--calcite-font-size--3)",  // 10px
+      "-2": "var(--calcite-font-size--2)",  // 12px
+      "-1": "var(--calcite-font-size--1)",  // 14px
+      0: "var(--calcite-font-size-0)",      // 16px
+      1: "var(--calcite-font-size-1)",      // 18px
+      2: "var(--calcite-font-size-2)",      // 20px
+      3: "var(--calcite-font-size-3)",      // 26px
+      4: "var(--calcite-font-size-4)",      // 32px
+      5: "var(--calcite-font-size-5)",      // 40px
+      6: "var(--calcite-font-size-6)",      // 48px
+      7: "var(--calcite-font-size-7)",      // 56px
+      8: "var(--calcite-font-size-8)",      // 64px
     },
     fontWeight: {
-      light: 300,
-      normal: 400,
-      medium: 500,
-      bold: 700
+      // assets/styles/_type
+      light: "var(--calcite-font-weight-light)",
+      normal: "var(--calcite-font-weight-normal)",
+      medium: "var(--calcite-font-weight-medium)",
+      bold: "var(--calcite-font-weight-bold)"
+    },
+    opacity: {
+      // assets/styles/global
+      disabled: "var(--calcite-ui-opacity-disabled)"
```

Comment:
Does this need to be themeable? Could we replace with `opacity-25` or `opacity-50`? Could we move this under `extend` to keep the existing opacity scale?

Reply:
I could see that. I many need to get you on the phone for when to use `extend`.

Reply:
I don't think we need to make our opacity level themeable, but it should reflect the current .4 value. We could probably update to just use opacity-50 without many folks even noticing though...

Reply:
Thanks @macandcheese. 👍🏾

Reply:
To be clear, if we go with `opacity-50`, we would just remove the custom stuff from the tailwind config and just use `opacity-50` in component styles, yeah?

Reply:
It'd still be nice to have as a something that can be reused without a raw opacity definition in a file, but, not sure if this opacity.disabled could just use an underlying tailwind variable or what

Reply:
I dont think this needs to be themeable. Keeps it real simple. Why cant we keep it at 0.4 ?

Reply:
Here are some options if I am following the discussion correctly:

Extending the default opacity set:
```
theme: {
  extend: {
     disabled: '0.4'
  }
}
```
But if the goal is constrain the choices you might want to omit `opacity-50` since it is relatively close to `opacity-40`.
```
theme: {
  opacity: {
    '0': '0',
    '25': '0.25',
    'disabled': '0.4',
    '75': '0.75',
    '100': '1',
  },
}
```
And then you might want to use `opacity-disabled` outside of a disabled state.
```
<div class='overlay opacity-disabled'>
```
So we'll add `opacity-40`.
```
theme: {
  opacity: {
    '0': '0',
    '25': '0.25',
    '40': '0.4',
    'disabled': '0.4',
    '75': '0.75',
    '100': '1',
  },
}
```
But now we've got two utilities with the same value. So the following are equivalent.
```
<button class='disabled:opacity-disabled'>
<button class='disabled:opacity-40'>
```
So if you assume that `opacity: 40` is unlike to change for disabled states you could then omit `opacity-disabled`:
```
theme: {
  opacity: {
    '0': '0',
    '25': '0.25',
    '40': '0.4',
    '75': '0.75',
    '100': '1',
  },
}
```

Reply:
I think the idea with naming / aliasing one of these to `opacity-disabled`, is that when a component is being developed that needs to have a disabled state set, it's easier to reach for a "named value". We can just make it 0.5 so it's on the scale, I think the idea is that we just alias that to be `opacity-disabled`. Within our own components, I don't think we use opacity treatments in a way that we need to change the defaults. IMO